_data/config-manual.yml

```diff
@@ -173,17 +176,17 @@ filter/preventBypassing:
 filter/removeBlockedDNS:
   layered: true
   manual: |
-    Should a DNS response contain IP addresses which would not be allowed if connected to, the Portmaster will remove these answers in order to make it more likely for a connection to succeed within the permitted parameters.
+    Should a DNS response contain multiple IP addresses, and some of them would not be allowed to be connected to, the Portmaster will remove these answers in order to make it more likely for a connection to succeed within the permitted parameters.
 
 filter/removeOutOfScopeDNS:
   layered: true
   manual: |
-    [DNS Rebinding](https://en.wikipedia.org/wiki/DNS_rebinding) attacks allow an attacker to circumvent security policies. This feature blocks DNS Rebinding attacks on local systems.
+    [DNS Rebinding attacks](https://en.wikipedia.org/wiki/DNS_rebinding) allow an attacker to circumvent security policies. This feature blocks DNS Rebinding attacks on local systems.
 
 filter/serviceEndpoints:
   layered: true
   manual: |
-    If you need to accept incoming connections, try to narrow down from where you need to accept connections. A great way to start is to only accept connections from an organization via its AS number ([find it here](https://www.ultratools.com/tools/asnInfo)) or from one to a few countries.
```

Comment:
I'd find that link very helpful.

Reply:
that link was broken on my end

Reply:
https://github.com/safing/docs/pull/12/commits/69c270480090301f4c84a057a0c960399e1ed20c

but please add a working one if you can

Reply:
I found this one: https://bgpview.io/